public/js/actions/subscriptions.js

```diff
@@ -104,8 +107,10 @@ export function _createSubscription({dispatch, productId,
         $xhr.responseJSON.error_response) {
       var allErrors = $xhr.responseJSON.error_response.__all__;
       if (allErrors && arrayHasSubString(allErrors, 'already subscribed')) {
-        dispatch(appActions.error('Subscription creation failed', {
```

Comment:
the old approach would log the message 'subscription creation failed' which I think is still important